playpass.toml

```diff
@@ -1,2 +1,3 @@
 # Your game's unique identifier. This should never change.
-game_id = "3e7645a1-c7e1-401b-a36f-cbdf7624d13d"
+game_id = "YOUR_GAME_ID"
```

Comment:
if we reset this game_id it's going to end up breaking the CI for showing the base template demo page linked in the readme.  Right now when you create a game using the playpass CLI this file gets wiped and generated so you'd end up with your game id regardless of what's in here.